src/app.test.js

```diff
@@ -59,8 +80,9 @@ test("shows new item in todo list when added", () => {
   ).toBeInTheDocument();
 });
 
-test("removes an item from todo List when user clicks Delete button", () => {
+test("removes an item from todo List when logged in user clicks Delete button", () => {
```

Comment:
I don't think we need to add "logged in" to any of the test case names as it is obvious that none of the Todo App functionality can be used without "logging in" first